packages/web/components/JournalyEditor/Toolbar/Toolbar.tsx

```diff
@@ -65,14 +66,16 @@ const Toolbar = ({
       setViewportsDiff(visualViewport.offsetTop)
     }
 
-    onVisualViewportChange()
+    const debouncedOnVisualViewportChange = useDebouncedCallback(onVisualViewportChange, 500)
```

Comment:
Calling a hook inside a `useEffect` callback is invalid, which contributes to the case for this being a simple function rather than a hook.